pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -65,7 +65,7 @@ type ActualStateOfWorld interface {
 	// returned.
 	// If no node with the name nodeName exists in list of attached nodes for
 	// the specified volume, an error is returned.
-	SetVolumeMountedByNode(volumeName v1.UniqueVolumeName, nodeName types.NodeName, mounted bool) error
+	SetVolumeMountedByNode(volumeName v1.UniqueVolumeName, nodeName types.NodeName, mounted bool, forceUnmount bool) error
```

Comment:
function's comment should be amended to include the new forceUnmount parameter. since the comment pretends atm that there's only one bool